mentors/php/v.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username": "nizey",
```

Comment:
`nizey` does not resolve as a github name, in other words going to https://github.com/nizey results in 404.  Not sure why it would do that, except that there may be a typo?  Do private profiles do this? @kytrinyx ?

Reply:
I don't think there are private profiles on GitHub.

Reply:
Thought maybe it was something new.  :sweat: